cmd/ipfs-cluster-ctl/main.go

```diff
@@ -89,10 +89,10 @@ func main() {
 	app.Description = Description
 	app.Version = Version
 	app.Flags = []cli.Flag{
-		cli.StringFlag{
+		cli.StringSliceFlag{
 			Name:  "host, l",
-			Value: defaultHost,
-			Usage: "Cluster's HTTP or LibP2P-HTTP API endpoint",
+			Value: &cli.StringSlice{defaultHost}, 
+			Usage: "Cluster's HTTP or LibP2P-HTTP API endpoint. To provide multiple hosts: --host a --host b",
```

Comment:
There is a non-written style rule that we provide lists to arguments as comma-separated values rather than using the StringSlice way. This means you keep the String flag, but do a `strings.Split` with the value when processing it.